.github/workflows/python.yml

```diff
@@ -74,9 +74,10 @@ jobs:
           ${{ runner.os }}-pip-${{ matrix.python-version }}-
   
     - name: Install pip Dependencies
-      run: pip install numpy py4j wheel
-      run: pip install scipy
-      run: pip install sklearn
+      run: |
+        pip install numpy py4j wheel
+        pip install scipy
+        pip install sklearn
```

Comment:
while merging change this to 

`run: pip install numpy py4j wheel scipy sklean`